docs/languages/powershell.md

```diff
@@ -0,0 +1,128 @@
+
+ Alternatively you can install from PowerShell command line
+ this works on all platforms
+
+    code --install-extension ms-vscode.powershell
+
+## Example Scripts
+
+Example scripts are included with the extension and can be found at the following Path.
+
+    C:\Users\<yourusername>\.vscode\extensions\ms-vscode.PowerShell-<version>\examples
+
+To open or view the examples in Visual Studio Code run the following from your PowerShell command prompt
+
+    code (Get-ChildItem $Home\.vscode\extensions\ms-vscode.PowerShell-*\examples)[-1]
+
+If using the Insiders edition
+
+    code-insiders (Get-ChildItem $Home\.vscode\extensions\ms-vscode.PowerShell-*\examples)[-1]
+
+You can also Open them with ctl/cmd+shift+p and typing __PowerShell: Open Examples Folder__.
+
+![Open PowerShell Examples](./images/powershell/pwshExamples.png)
+
+## Major Features
+
+* Syntax Highlighting
+* Code Snippets
+* Intellisense for cmdlets and more
+* Rule-Based analysis provided by PowerShell Script Analyzer
+* Go to definition of cmdlets and variables
+* Find references of cmdlets and variables
+* Document and Workspace symbol discovery
+* Run slected selection of PowerShell code using F8
+* Launch online help for the symbol under the cursor using ctl+F1
+* Local script debugging and basic interactive console support
+
+## Plaster
+
+ Plaster is a template-based file and project generator written in PowerShell. It's Purpose is to streamline the creation of PowerShell module projects, Pester tests, DSC configurations and more.
+ See the [Github Plaster repo](https://github.com/PowerShell/Plaster) for more information.
+
+ [Cmdlet Documentation](https://github.com/PowerShell/Plaster/blob/master/docs/en-US/Plaster.md)
+
+## Debugging
+
+Check out the ![HSG](./images/powershell/HSGavatar.png) Hey, Scripting Guy blog Posts two part series on debugging
+
+Written by [@keithHill](https://twitter.com/r_keith_hill)
+
+[Debugging PowerShell script in Visual Studio Code – Part 1](https://blogs.technet.microsoft.com/heyscriptingguy/2017/02/06/debugging-powershell-script-in-visual-studio-code-part-1/)
+
+[Debugging PowerShell script in Visual Studio Code – Part 2
+](https://blogs.technet.microsoft.com/heyscriptingguy/2017/02/13/debugging-powershell-script-in-visual-studio-code-part-2/)
+
+## Settings Reference
+
+You can customize VSCode settings from the file menu by selecting preferences then clicking settings. You can also click the gear Icon
+![codeGear](./images/powershell/codeGear.png) located in the lower left corner of the activity bar.
+
+This will launch the settings.json file, in VSCode versions prior to 1.27.1. , where the VSCode team has introduced a settings GUI, which will be the new default interface for customizing settings.
+
+You can still launch the JSON settings file by using the command __Open Settings (JSON)__ or by changing the default settings editor with the ```"workbench.settings.editor"```
```

Comment:
The settings.json file should be referred to consistently as either "settings.json" or whatever else is best to call it to prevent confusion. We should also link to an explanation of the settings.json file provided by the VSCode docs

Reply:
Might also be worth an explanation of the command palette. Maybe the word [command](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) should be a link to the Command Palette documentation?

Reply:
Also ` ```workbench.settings.editor``` ` only needs single backticks: `` `workbench.settings.editor` ``. (Yes, escaping those backticks in my example to embed them in backticks is witchcraft...)